tools/build_project.py

```diff
@@ -371,10 +376,10 @@ def main():
         yaml.dump(manifest["gbl"], f)
 
     # Next, generate a chip-specific project from the modified base project
-    print(f"Generating project for {manifest['device']}")
+    LOGGER.info(f"Generating project for {manifest['device']}")
 
     # fmt: off
-    subprocess.run(
+    slc_result = subprocess.Popen(
```

Comment:
I don't think you can mix `stdout`/`stderr` with `capture_output`, so for some cmds, it might not be possible to use it. Would run the risk of some stuff in `stderr` (non-hard-fail) being included?

Reply:
I think we only use it in a single spot, for `git` (never assumed to fail) and `jq` in the new PR. I can just switch to the jq Python bindings to avoid using subprocess: https://pypi.org/project/jq/